.github/workflows/ci.yml

```diff
@@ -6,24 +6,13 @@ env:
   CI: true
 jobs:
   build:
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        # Available OS's: https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners
-        os: [ubuntu-18.04]
-        node-version: [14.x, 12.x]
+    runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v2-beta
-      with:
-        node-version: ${{ matrix.node-version }}
-    - name: Cache node modules
-      uses: actions/cache@v1.1.2
```

Comment:
I could've upgraded to `actions/cache@v2` but `actions/cache@v3` requires node 16, so we couldn't use that. Using `cache: npm` should be good enough though for now.